.github/workflows/on_push_main.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - "main"
 
+  workflow_call:
```

Comment:
This looks like a mistake - if it isn't, please add a comment explaining it to the reader